tests/args_test.go

```diff
@@ -33,12 +34,6 @@ func aTestFunc(argParse interpreter.ArgParser, output io.Writer) (interpreter.Re
 	return nil, nil
 }
 
-type Output struct{}
```

Comment:
the prototype for the `script.Func`s are `type Func func(args ArgParser) (Retval, error)` so Im confused :/